apps/charterafrica/src/payload/fields/accordion.js

```diff
@@ -53,7 +53,18 @@ function accordion({ richTextConfig, overrides = {} } = {}) {
             required: true,
             localized: true,
             admin: {
-              elements: [],
+              elements: [
+                "h1",
+                "h2",
+                "h3",
+                "h4",
+                "h5",
```

Comment:
```suggestion
```

Don't think we need headings in the accordion... Given the accordion title is `h5`, may be we can allow `h6` if need be.